scania-digital-design-system/tegel
# test(stepper): added unit test

## **Describe pull-request**  
Added unit test for Stepper component

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3042](https://tegel.atlassian.net/browse/CDEP-3042)    

## **How to test**  
1. Take a look at the variations tested 
2. Check if the playwright test is passing (on your local or in PR)
3. Try to change something in index.html, like props - test should fail


## **Checklist before submission**




[CDEP-3042]: https://tegel.atlassian.net/browse/CDEP-3042?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
